### DISCUSSION TITLE
댓글 불러오기 관련

### DISCUSSION THREAD

**Discussion by enyo9rt** [2022-Harmony/NewsCommunity-bFinal#73] [General]
지금 댓글 가져오는 함수는 [댓글 작성 정보->작성 정보와 연결된 회원 정보 가져옴->회원과 연결된 프로필 가져옴->프로필 포함된 댓글 응답 dto만듦->반환] 이런 형태!
https://github.com/2022-Harmony/NewsCommunity-bFinal/blob/f58b050264f697acfd2203967f0664caffb1bc6f/src/main/java/com/teamharmony/newscommunity/comments/service/CommentService.java#L38-L53

회원 프로필 정보를 모두 보여주기 위해서는 크게 1.기본 프로필(닉네임,파일명,소개글) 2.프로필 이미지 url 이 두 가지가 필요
- 클라이언트에서 get을 두 번(댓글 쪽에서 getComments하고 회원 쪽에서 getProfileImgUrl) 하는 쪽으로 결정

**Comment by hyunzxn** [comment]
현재처럼 댓글 정보를 불러오고 프로필 이미지 링크를 얻기 위한 GET요청을 한 번 더 하란 말씀이시죵..?

**Reply by enyo9rt** [reply]
넵넵 일단은요... 더 좋은, 다른 방법이 있는지 모르겠어요

**Reply by hyunzxn** [reply]
음...일단은 이렇게 해보죠! 컨트롤러에서 String으로 return해주면 제가 변수에 한 번 담아서 써보는 식으로 해볼게요.
